src/Flatten.chpl

```diff
@@ -2,14 +2,14 @@ module Flatten {
   use ServerConfig;
 
   use AryUtil;
-  use SegmentedArray;
+  use SegmentedString;
   use ServerErrors;
   use SymArrayDmap;
   use CommAggregation;
   use Reflection;
   use ArkoudaRegexCompat;
   use CTypes;
-  use SegmentedArray only checkCompile, _unsafeCompileRegex;
```

Comment:
This is outside the scope of this PR but prob don't need both of these imports. I actually think there's a fair chance this is my fault

Reply:
I agree. I will go ahead and clean this up while it is in front of us.

Reply:
Removed the 2nd statement